lms/migrations/versions/5e029d1d99f9_keys.py

```diff
@@ -0,0 +1,44 @@
+"""
```

Comment:
I suggest ignoring the migrations here and just look at the model changes. It's easier to follow.